R/bs_sass.R

```diff
@@ -119,7 +132,7 @@ bootstrap <- function(
       htmlDependency(
         name = "bootstrap",
         version = get_exact_version(version),
-        src = dirname(out_file),
+        src = out_file_dir,
         stylesheet = basename(out_file),
         script = basename(js),
```

Comment:
Since we're already assuming this behavior, let's be explicit about it

```suggestion
  all_files = TRUE,
```